src/components/molecules/UserProfilePicture/UserProfilePicture.tsx

```diff
@@ -101,8 +105,12 @@ const UserProfilePicture: React.FC<UserProfilePictureProp> = ({
           className={avatarClassName}
           backgroundImage={pictureUrl}
           style={{ ...avatarStyle }}
-        />
-
+        >
+          {showNametags &&
+            (currentVenue?.showNametags ?? SHOW_AVATAR_NAMETAGS) && (
```

Comment:
You could make this line a little cleaner by extracting the boolean logic out into a `const` higher up in this component. Probably something like

```typescript
const shouldShowNameTags = showNametags && (currentVenue?.showNametags ?? SHOW_AVATAR_NAMETAGS);
```

Then here you could just have:
```suggestion
          {shouldShowNametags && (
```

Reply:
I'm not 100% what the best logic is here though.

Do you think the local `showNameTags` prop should override the venue-wide setting? My guess is that it was added so that we can force the name tags not to show in certain locations.

Reply:
And based on that, perhaps there is a better name we can use for this prop, that better conveys the fact that it controls whether the venue setting is respected/not. As it currently is, I might assume that by setting it to `true`, it should always display, so I would be confused when it didn't (because the venue setting was off)

Reply:
This is a good point. I vaguely recall implementing this to get around an issue of the nametags showing up in chat window. I've implemented the change you suggested here, but think about better terminology for the variables